### ISSUE TITLE
Add support for kVTCompressionPropertyKey_DataRateLimits

### ISSUE THREAD

**Issue by triwav** [shogo4405/lf.swift#71]
It be great to be able to set kVTCompressionPropertyKey_DataRateLimits as the average bitrate is often overrun and being able to set more firm limits on it would be beneficial.

**Comment by shogo4405** [comment]
@triwav  Hi. framework supported it.
Try here. kVTCompressionPropertyKey_DataRateLimits seems to adjust a FPS.
```swift
let bitrate:UInt32 = 160 * 1024 // bitPerSecond
let dataRateLimits:[Int] = [160 * 1024 / 8, 1] // [byte, second]
rtmpStream.videoSettings = [
    "bitrate": bitrate,
    "dataRateLimits": dataRateLimits,
]
```

**Comment by triwav** [comment]
Have you tested out this commit? Looking at this line: https://github.com/shogo4405/lf.swift/commit/277852cb2cfc52cb105b34c6130a4ba68da461c8#diff-eab61c0f1ecc489ad80ca77abb811cd6R81

You are checking the oldValue as well and returning if it was nil. This means the first time you set this property to something besides nil it should be ignored. I haven't actually tested this but I'm pretty sure that's what it will do.

**Comment by shogo4405** [comment]
oh... You're right. I fixed it.

**Comment by shogo4405** [comment]
released 0.3.4
